nix/cardano-services/deployments/handle-provider-deployment.yaml

```diff
@@ -66,7 +66,7 @@ spec:
           image: FIXME-ADD-LATEST-IMAGE-REFERENCE
           livenessProbe:
             httpGet:
-              path: /v1.0.0/health
+              path: /v1.0.0/live
```

Comment:
Could it be this change should be reflected on health checks of our `docker-compose` files?
I see CI is passing... I would have expected all containers to be unhealthy...

Could it be we are adding the `/live` endpoint **and** we are keeping the `/health`?

Thank you

Reply:
Yes, great catch! Yes, health is staying since it includes information about the service state. `live` and `ready` are binary responses

Reply:
Much better! We don't need to change `docker-compose` cfg as it's heath checks are not impacted.